docs/get-started/start-teku.md

```diff
@@ -125,6 +125,36 @@ Content-Length: 0
   </TabItem>
 </Tabs>
 
+## Exit Codes
+
+**Exit Code 1**
+
+This code indicates a scenario where Teku has exited with a fatal error, however restarting Teku without changes is a logical step in correcting it. This code represents the error is related to something external to Teku.
+
+Example: If the beacon chain controller has issues starting P2P services on a particular port, the system will exit with a fatal error code of 1. This could correct itself with a restart.
```

Comment:
... will exit with a fatal error code of 1. This could correct itself with a restart. = ... will exit with a fatal code 1 error. Restart the system to correct this.